content/blog/2023-05-30-may-curation-validation/contents.lr

```diff
@@ -0,0 +1,71 @@
+
+From ensuring deposited records abide by FAIR principles, to granular access control, InvenioRDM supports resource curation and validation with a variety of functionalities. Below are some of the key features that make the platform a great option for sharing and preserving your work:
+
+
+## Deposits:
+
+- Deposited records check all FAIR principles (Findability, Accessibility, Interoperability, Reusability)
+
+- Long-running record drafts for pre-publication edits
+
+- A wide variety of licensing options
+
+- Ability to set an embargo period for records or files with automated embargo removal at a specified date
+
+- Contextualization of records via secondary identifiers to publications, project sites, external data, and other associated research outputs
+
+- Citations in multiple formats provided for each record
+
+- Empowers communities to self-curate domain specific records
+
+- Ability to archive software via GitHub integration within records, allowing data to be linked to code [coming soon]
+
+- Usage metrics and analytics (COUNTER-compliant statistics gathering for record access and downloads) [coming soon]
+
+
+## Documentation:
+
+- Deposit agreement to ensure no sensitive data is uploaded
+
+- Extensive support documentation
+
+- Configurable terms-of-use for the repository
+
+
+## Metadata and Discovery:
+
+- Supports industry standards for interoperability; metadata is indexed using these standards
+
+- Ability to broker metadata to search and discovery services
+
+- Hand curation by repository administrators for metadata quality assurance
+
+- Versioning of all deposits, each with a permanent identifier (DOI), and one concept DOI for the record entity through all versions
+
+- Full-text searching to facilitate data discovery
+
+
+## Access, Security, and Quality Control:
+
+- Authentication configurable with institutional credentials or third-parties (e.g., ORCiD, GitHub)
+
+- Checksum for uploaded files ensure file integrity throughout each file's lifetime 
+
+- Restricted access capability
+
+- Can be configured for secure storage
+
+- Automated periodic file audits
+
+- InvenioRDM instances can be reviewed and certified by CoreTrustSeal (https://www.coretrustseal.org/)
```

Comment:
Are we sure of this one? it might need extra integrations as with OAIS complaint platforms.

Reply:
I think we can remove line 73:

`InvenioRDM instances can be reviewed and certified by CoreTrustSeal (https://www.coretrustseal.org/)`

This would have to be done on the instance level and is dependent on local environment, configuration, and curation policies.